admin/resources/js/components/dashboard/Dashboard.tsx

```diff
@@ -45,9 +45,24 @@ export const Dashboard: React.FC<{
 }> = ({ menuItems, contentRoutes }) => {
   const content = useRouter(contentRoutes);
   return (
-    <div>
-      <SideMenu items={menuItems} />
-      <div>{content}</div>
+    <div
+      data-h2-flex-grid="b(stretch, contained, flush, none)"
+      style={{
+        minHeight: "100vh",
+      }}
+    >
+      <div
+        data-h2-flex-item="b(1of1) l(2of12)"
```

Comment:
I think this should be `s(2of12)`, or at most `m`. With the `l` media query it doesn't show up side-by-side in storybook, even at max width.